go.mod

```diff
@@ -47,14 +47,14 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/mock v1.6.0
 	github.com/golang/snappy v0.0.4
-	github.com/google/go-cmp v0.5.6
+	github.com/google/go-cmp v0.5.7
 	github.com/google/uuid v1.3.0
 	github.com/google/wire v0.5.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/gosimple/slug v1.9.0
 	github.com/grafana/cuetsy v0.0.0-20211119211437-8c25464cc9bf
 	github.com/grafana/grafana-aws-sdk v0.9.1
-	github.com/grafana/grafana-plugin-sdk-go v0.123.0
+	github.com/grafana/grafana-plugin-sdk-go v0.124.1-0.20220127163549-4ffddbc40722
```

Comment:
@toddtreece do we plan to release the SDK bump soon?

Reply:
yeah, i will do that tonight. i was waiting to see if we needed the 8.3.6 backport

Reply:
https://github.com/grafana/grafana/pull/44685